src/librustc_codegen_ssa/back/link.rs

```diff
@@ -1500,8 +1500,8 @@ fn linker_with_args<'a, B: ArchiveBuilder<'a>>(
     cmd.optimize();
 
     // OBJECT-FILES-NO, AUDIT-ORDER
-    // Pass debuginfo flags down to the linker.
-    cmd.debuginfo();
+    // Pass strip flags down to the linker.
```

Comment:
```suggestion
    // Pass debuginfo flags down to the linker.
```

Reply:
Awful sorry for so many omissions.

Reply:
IMHO,
```
    // Pass debuginfo and strip flags down to the linker.
```
is more accurate.

Reply:
Another question, do I need to manually squash commits. Or bors would do that automatically for me?

Reply:
bors will applies commits exactly, without squashing

Reply:
Thanks. I squashed the commits maunally.